scheduler/utilization_based_host_allocator.go

```diff
@@ -151,8 +151,14 @@ func evalHostUtilization(ctx context.Context, d distro.Distro, taskGroupData Tas
 	}
 	freeHostDur := time.Since(startAt)
 
+	var roundDown bool
```

Comment:
To simplify, could just set roundDown to false immediately, and only switch to true if RoundRule >= 0 (or vice versa)